tethne/dao/tethnedao.py

```diff
@@ -0,0 +1,63 @@
+import psycopg2
```

Comment:
I'd rather not add this to the requirements, since most users will not use this module. Can we add a try/except block here that attempts to import, and then raises a RuntimeError if psycopg2 is missing?